bench/src/main/scala/geotrellis/raster/GenericRasterBench.scala

```diff
@@ -0,0 +1,62 @@
+package geotrellis.raster
+
+import geotrellis.bench.init
+import org.openjdk.jmh.annotations._
+import org.openjdk.jmh.infra.BenchmarkParams
+
+import scala.util.Random
+
+
+@BenchmarkMode(Array(Mode.AverageTime))
+@State(Scope.Thread)
+@Threads(4)
```

Comment:
Always 4?

Reply:
Can't remember why I did that. I'll either remove or comment why the reason.